modules/ROOT/pages/_partials/sync/js-client-realtime-updates.adoc

```diff
@@ -1,3 +1,63 @@
 = Realtime Updates
 
-TODO
\ No newline at end of file
+Subscriptions are a core concept of GraphQL and one which allow us to enable real time updates and synchronisation of data.
```

Comment:
maybe it's a good idea to explain what subscriptions can do and can't do, to make it clear what use cases it supports.

Reply:
+100. Plus mention how other functionalities we provide interact with subscriptions.
How offline/reconnect stuff works etc.